dashboard/app/controllers/courses_controller.rb

```diff
@@ -16,10 +16,17 @@ class CoursesController < ApplicationController
   authorize_resource class: 'UnitGroup', except: [:index], instance_name: 'unit_group', id_param: :course_name
 
   def new
-    @course_families_course_types = UnitGroup.family_names.map do |cf|
-      ug = CourseOffering.find_by(key: cf).course_versions.first.content_root
-      [cf, {instruction_type: ug.instruction_type, instructor_audience: ug.instructor_audience, participant_audience: ug.participant_audience}]
-    end.to_h
+    @versioned_course_families = []
+    @course_families_course_types = []
+    UnitGroup.family_names.map do |cf|
+      co = CourseOffering.find_by(key: cf)
+      first_cv = co.course_versions.first
+      ug = first_cv.content_root
+      @versioned_course_families << cf unless first_cv.key == 'unversioned'
+      @course_families_course_types << [cf, {instruction_type: ug.instruction_type, instructor_audience: ug.instructor_audience, participant_audience: ug.participant_audience}]
+    end
+
+    @course_families_course_types.to_h
```

Comment:
can you remind me, does the return value for this method get used somewhere? here and below. 

the place I would start looking is https://guides.rubyonrails.org/action_controller_overview.html, but I didn't see anything definitive. if you can't find a way that it's used, and you feel good about your test coverage, I would suggest removing it.

Reply:
I wasn't actually trying to return `@course_families_course_types` I just needed for follow up after setting it up to convert it to a hash. I'm going to leave this and merge it but if you have thoughts when you get back I'm happy to follow up

Reply:
This returns a hash, but does not convert the originals object into a hash, so this statement ultimately looks like it has no effect.

Reply:
Oh I see what I missed. Ok thanks I will look into this.

Reply:
Fixing it here: https://github.com/code-dot-org/code-dot-org/pull/46239